action.yml

```diff
@@ -4,21 +4,46 @@ description: 'Download the Kernel source to build a kmod or dmks in actuated'
 branding:
   icon: 'arrow-right-circle'
   color: 'gray-dark'
+  
+inputs:
+  cache:
+    description: Whether to enable caching for the downloaded sources
+    required: false
+    default: 'false'
```

Comment:
Default to `false`; do we want to enable caching by default?

Reply:
Good question. That seems to be the default for certain actions, so you could do?

Reply:
I think it might make sense to enable by default, since it will save time.

Reply:
Enabled!